components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -78,9 +78,17 @@
     private String passiveSTSWReply;
     private String openid;
     private String[] claimUris;
+    private List<InboundAuthenticationRequestConfig> customInbountAuthenticatorConfigs;
+    private List<String> standardInboundAuthTypes;
 
     public ApplicationBean() {
 
+        standardInboundAuthTypes = new ArrayList<String>();
+        standardInboundAuthTypes.add("oauth2");
+        standardInboundAuthTypes.add("wstrust");
+        standardInboundAuthTypes.add("samlsso");
+        standardInboundAuthTypes.add("openid");
+        standardInboundAuthTypes.add("passivests");
```

Comment:
Do we need to have these standard types in the ApplicationBean. Having it in the DAO level is should be enough. At this level we can treat all as generic config types right ?